Source/Diagnostics/WarpXOpenPMD.H

```diff
@@ -245,6 +251,7 @@ private:
 
   bool m_OneFilePerTS = true;  //! write in openPMD fileBased manner for individual time steps
   std::string m_OpenPMDFileType = "bp"; //! MPI-parallel openPMD backend: bp or h5
+  std::string m_OpenPMPoptions = "{}"; //! JSON option string for openPMD::Series constructor
```

Comment:
why m_OpenPMPoption, instead of m_OpenPMDOption?  Anything special with PMP?

Reply:
Oh no, that's a typo. Good catch, thanks!